### PR TITLE
feat(PAYG-1374): Remove user info from get payment response

### PR DESCRIPTION
**Pull request by tl-tai-tang**
**Description:** 
Remove user info from the `GET /payment` response.

**Changelog:**
https://docs.truelayer.com/changelog/removal-of-user-info-in-payment-and-mandate-response

**Updated documentation**
https://docs.truelayer.com/reference/get-payment-1
https://docs.truelayer.com/reference/get-mandate

### CODE REVIEW

**Comment by tl-tai-tang** [comment]
PR is ready for review but not ready for merging, Thanks

File: src/TrueLayer/Payments/Model/PaymentUser.cs [hunk 1]
```diff
@@ -3,5 +3,5 @@ namespace TrueLayer.Payments.Model
     /// <summary>
     /// Represents an end user
     /// </summary>
-    public record PaymentUser(string Id, string? Name = null, string? Email = null, string? Phone = null);
+    public record PaymentUser(string Id);
```
Comment (tl-mauro-franchi) [inline src/TrueLayer/Payments/Model/PaymentUser.cs:6]: This is a breaking change for the consumers of the SDK as the data is currently returned in the `GetPaymentResponse` object. Can you please link the API documentation in the PR and link any public change log stating that the fields get removed? Or, are we sure that those fields never received any value?

**Review by tl-mauro-franchi — CHANGES_REQUESTED** [review]

File: src/TrueLayer/Payments/Model/PaymentUser.cs (hunk 1 above)
Comment (tl-tai-tang) [reply src/TrueLayer/Payments/Model/PaymentUser.cs:6]: The change log and API doc update are on staging. Will share the links here as soon as they are available in production. 

Currently, those fields may contain a value but we would like to stop returning them. The data should be available to the client internally as it's submitted to us by the client as part of the payment creation. 

 This is done to reduce sensitive information exchanges

**Review by tl-mauro-franchi — APPROVED** [review]
Approved based on the comment above. Please remember to raise a new PR and update the [changelog](https://github.com/TrueLayer/truelayer-dotnet/blob/main/CHANGELOG.md) **after** this PR is merged and published. Thank you.

**Pull request merged**